utils/sms-tool/Makefile

```diff
@@ -0,0 +1,39 @@
+# 2017 - 2021 Cezary Jackiewicz <cezary@eko.one.pl>
+# 2014 lovewilliam <ztong@vt.edu>
+# sms tool for various of 3g modem
```

Comment:
not 4g?

Reply:
Residue from the beginning of the application. Program works with 3G / 4G / 5G modems.